app/uk/gov/hmrc/economiccrimelevyregistration/views/contacts/AddAnotherContactView.scala.html

```diff
@@ -37,15 +37,11 @@
         @errorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "yes")))
     }
 
-    <h1 class="govuk-heading-l">@messages("addAnotherContact.heading")</h1>
-
-    <p class="govuk-body">@messages("addAnotherContact.legend")</p>
-
```

Comment:
Why did you remove those?

Reply:
Because it shouldn't be there. When you set a legend to asPageHeading, it creates the H1 for you.

In addition, hinttext takes the place of that paragraph.